### PR TITLE
Rounded Post/Comment Editor Design

### PR DESCRIPTION
**Pull request by Sjmarf**

### CODE REVIEW

File: Mlem/App/Views/Pages/PostEditor/PostEditorView+Logic.swift [hunk 1]
```diff
@@ -121,4 +121,26 @@ extension PostEditorView {
...
+    
+    func saveFocusState() {
+        if contentTextView.isFirstResponder {
+            lastFocusedField = .content
+        } else if titleTextView.isFirstResponder {
+            lastFocusedField = .title
+        } else {
+            lastFocusedField = nil
+        }
+        print(lastFocusedField)
```
Comment (EricBAndrews) [inline Mlem/App/Views/Pages/PostEditor/PostEditorView+Logic.swift]: *nit*

Still needed?

**Review by EricBAndrews — CHANGES_REQUESTED** [review]
Code looks good!

I see this when I open the post composer:

IMG_EE1698E64C44-1

**Comment by Sjmarf** [comment]
Nice catch, fixed 👍 Strangely this issue didn't happen on my physical device, but I could reproduce it on a simulator.

**Review by EricBAndrews — APPROVED** [review]
LGTM! 🚀

**Pull request merged**